### DISCUSSION TITLE
More details about Gutenberg Phase4-Multilingual?

### DISCUSSION THREAD

**Discussion by eddybase** [WordPress/gutenberg#59260] [Q&A]
## What problem does this address?

the project is in the second phase of a four-phase process that will touch every piece of WordPress -- Editing, Customization (which includes Full Site Editing, Block Patterns, Block Directory and Block based themes), Collaboration, and **Multilingual**

Can I assume that we'll no more need to use PolyLang or similar plugine?

We have some related developent in the near future and we dont' want to reinvent the wheel.

Thanks for more details shared.

**Comment by michaelgiraldo** [comment]
Multilingual support is key. Looking at existing solutions, they all fall short. Native multilingual support is critical.
